packages/dcos-integration-test/extra/test_helpers.py

```diff
@@ -1,3 +1,4 @@
 from pkgpanda.util import load_json
 
 dcos_config = load_json('/opt/mesosphere/etc/expanded.config.json')
+expanded_config = load_json('/opt/mesosphere/etc/expanded.config.json')
```

Comment:
Where did this change come from? I don't see it in any of the PRs or the cherry-pick from master, but I guess it's included in https://github.com/dcos/dcos/pull/1605/commits/1b83cd18224b4617b5ac99104d072e3ac2a5cab1 because it's required by the cherry-pick? Not obvious from the PR description.
cc: @mellenburg to confirm the safety/necessity of this change. I'm pretty sure we always had expanded.config.json in 1.9.x